readme.md

```diff
@@ -154,6 +154,11 @@ console.log(redact.restore(o) === o) // true
 console.log(o) // { a: 1, b: 2 }
 ```
 
+#### `strict` – `Boolean` - `[true]`
+The `strict` option, when set to `true`, will cause the redactor function to throw an error when it encounters
+a primitive type. When `strict` is set to `false`, the redactor function will return the primitive value without
```

Comment:
This is not completely correct. it will throw if instead of an object it finds a primitive.